setup.py

```diff
@@ -34,7 +34,7 @@
 
 setup(
     name='keria',
-    version='0.2.0-dev3',  # also change in src/keria/__init__.py
```

Comment:
Just a note that you also need to change it in the Makefile

Reply:
done!